python/lsst/daf/butler/script/certifyCalibrations.py

```diff
@@ -78,8 +78,13 @@ def certifyCalibrations(
     if not search_all_inputs and registry.getCollectionType(input_collection) is CollectionType.CHAINED:
         input_collection = next(iter(registry.getCollectionChain(input_collection)))
 
-    refs = set(registry.queryDatasets(dataset_type_name, collections=[input_collection]))
-    if not refs:
-        raise RuntimeError(f"No inputs found for dataset {dataset_type_name} in {input_collection}.")
+    with butler._query() as query:
+        results = query.datasets(dataset_type_name, collections=input_collection)
```

Comment:
The default for find-first has changed, so this is a behavior change.  But I think it's an unambiguous improvement, because in the past, if the query had returned multiple results for the same data ID, the `certify` call later would have choked on them.